docs/apis/webhooks/sharepoint-webhooks-using-azd-template.md

```diff
@@ -0,0 +1,178 @@
+---
+title: Create Azure functions for SharePoint webhooks using an azd template
+description: Use Azure Developer cli (azd) to deploy an Azure function app that connects to your SharePoint Online tenant, to register and manage webhooks, and process the notifications from SharePoint.
+ms.date: 02/17/2025
+ms.localizationpriority: low
+---
+# Azure functions for SharePoint webhooks using azd
+
+[Azure Developer CLI (azd)](https://aka.ms/azd) is an open-source tool that accelerates provisioning and deploying app resources in Azure.  
+This article uses the [Azure function app for SharePoint webhooks public template](https://github.com/Azure-Samples/azd-functions-sharepoint-webhooks) to deploy an Azure function app that connects to your SharePoint Online tenant, to register and manage [webhooks](overview-sharepoint-webhooks.md), and process the notifications from SharePoint.
+
+## Prerequisites
+
+- [Node.js 20](https://www.nodejs.org/)
+- [Azure Functions Core Tools](/azure/azure-functions/functions-run-local?pivots=programming-language-typescript#install-the-azure-functions-core-tools)
```

Comment:
Remove the deep link unless it's absolutely necessary. In this case:
- the install hash isn't necessary
- the language-specific `pivot` shouldn't be included

ref: MSDOCS & MSLEAN content & markdown formatting rules